client/src/components/ModeToggle.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faLightbulb, faMoon } from '@fortawesome/free-solid-svg-icons'
+import { useTheme } from '../contexts/ThemeContextSet'
+
+export default function ModeToggle() {
```

Comment:
"ThemeToggle" would be more consistent.  Yes it's called "Dark Mode" or "Light Mode" to the user, but most of the rest of the code calls it "theme" so you should use that consistently